sdk/avs/Microsoft.Azure.Management.Avs/src/Microsoft.Azure.Management.Avs.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Avs</PackageId>
     <Description>Provides developers with libraries for Azure VMware Solution.</Description>
-    <Version>1.0.0-preview.1</Version>
+    <Version>1.0.0</Version>
```

Comment:
You will need to make a change to at least the AssemblyFileVersion in AssemblyInfo.cs - the file version needs to be monotonically increasing, or some kinds of installation processes (like msis) will not recognize that this assembly is different from the preview.

Reply:
I set AssemblyFileVersion in 8835f83.